env.example

```diff
@@ -4,6 +4,16 @@ NODE_ENV=development
 # PORT is the port used by the web server
 PORT=3000
 
```

Comment:
Haha sorry, I know it's pedantic, but can you get rid of the extra newlines on line 6 and 15?